payments.py

```diff
@@ -5,11 +5,17 @@
 import re
 import time
 from flask import Flask, jsonify, request, make_response, Response, json, url_for
+from flask_sqlalchemy import SQLAlchemy
 
 # Create Flask application
 app = Flask(__name__)
 #app.config['LOGGING_LEVEL'] = logging.INFO
 
+# DB Config
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///data/dev.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
```

Comment:
yeah, makes sense. i'll add this to your existing module __init__ once merged in